features/zoeken.feature

```diff
@@ -1,51 +1,76 @@
+
+    Voorbeeld: zoek WOZ-object met postcode en huisnummer en huisnummertoevoeging van de aanduiding
+      Gegeven de LV WOZ kent een WOZ-object met objectnummer "000500000001"
+      En de aanduiding WOZ-object heeft postcode "8000GB" en huisnummer 1 en huisletter "a" en huisnummertoevoeging "bis"
+      Als ik een WOZ-object zoek met /wozobjecten?postcode=8000GB&huisnummer=1&huisnummertoevoeging=bis
+      Dan bevat het antwoord het WOZ-object met identificatie "000500000001"
+
+    Voorbeeld: zoek WOZ-object met postcode en huisnummer en huisletter en huisnummertoevoeging van de aanduiding
+      Gegeven de LV WOZ kent een WOZ-object met objectnummer "000500000001"
+      En de aanduiding WOZ-object heeft postcode "8000GB" en huisnummer 1 en huisletter "a" en huisnummertoevoeging "bis"
+      Als ik een WOZ-object zoek met /wozobjecten?postcode=8000GB&huisnummer=1&huisletter=a&huisnummertoevoeging=bis
+      Dan bevat het antwoord het WOZ-object met identificatie "000500000001"
+
+    Voorbeeld: zoek WOZ-object met postcode en huisnummer en onjuiste huisletter
+      Gegeven de LV WOZ kent een WOZ-object met objectnummer "000500000001"
+      En de aanduiding WOZ-object heeft postcode "8000GB" en huisnummer 1 en huisletter "a" en huisnummertoevoeging "bis"
+      Als ik een WOZ-object zoek met /wozobjecten?postcode=8000GB&huisnummer=1&huisletter=b
+      Dan bevat het antwoord niet het WOZ-object met identificatie "000500000001"
+
+    Voorbeeld: zoek WOZ-object met postcode en huisnummer en onjuiste huisnummertoevoeging
+      Gegeven de LV WOZ kent een WOZ-object met objectnummer "000500000001"
+      En de aanduiding WOZ-object heeft postcode "8000GB" en huisnummer 1 en huisletter "a" en huisnummertoevoeging "bis"
+      Als ik een WOZ-object zoek met /wozobjecten?postcode=8000GB&huisnummer=1&huisnummertoevoeging=andere
+      Dan bevat het antwoord niet het WOZ-object met identificatie "000500000001"
 
     Voorbeeld: zoek WOZ-object met nummeraanduidingIdentificatie van de aanduiding
-      Gegeven de LV WOZ kent een WOZ-object met objectnummer "051812345678"
-      En de aanduiding WOZ-object heeft nummeraanduidingIdentificatie "0518200000842215"
-      Als ik een WOZ-object zoek met /wozobjecten?nummeraanduidingIdentificatie=0518200000842215
-      Dan bevat het antwoord het WOZ-object met identificatie "051812345678"
+      Gegeven de LV WOZ kent een WOZ-object met objectnummer "000500000001"
+      En de aanduiding WOZ-object heeft nummeraanduidingIdentificatie "0005200000009179"
+      Als ik een WOZ-object zoek met /wozobjecten?nummeraanduidingIdentificatie=0005200000009179
+      Dan bevat het antwoord het WOZ-object met identificatie "000500000001"
+
+  Regel: zoeken met huisletter en huisnummertoevoeging is case-insensitive
+
+    Voorbeeld: zoek WOZ-object met postcode en huisnummer en huisletter en huisnummertoevoeging in andere casevan de aanduiding
```

Comment:
casevan => case van?
postcode is wel case sensitive? Ik weet niet of het logischer is om zoeken met postcode + huisnummer + varianten geheel case insensitive te maken